src/extra/validator/date.js

```diff
@@ -0,0 +1,45 @@
+window.Parsley.addValidator('date', {
+    validateString: function(value, format) {
+        if (! value) {
+            return true;
+        }
+      
+        var date = moment(value, format, true);
+        return date.isValid();
+    },
+    priority: 256,
+});
+
+window.Parsley.addValidator('datebeforenow', {
+    validateString: function(value, format) {
+        if (! value) {
+            return true;
+        }
+        
+        if (! date.isValid()) {
+            return true;
```

Comment:
Shouldn't that be `return false`??

Reply:
No, it must return `true` because you have also to add `data-parsley-date`, or you'll get an erroneous message. That's because I was trying to launch `data-parsley-date` inside the other two validators.

Reply:
Think it as `required` field. If another validator gets an empty value, it returns `true`.

Reply:
Best return `true`. `data-parsley-date` should simply have a higher priority. If you have both `data-parsley-date` and `datebeforenow`, you will get different error messages for an invalid date and a valid date that is after now. If you only have `datebeforenow`, then you will get a single error message and you will get it for an invalid date or a date after now.

Reply:
> If you have both `data-parsley-date` and `datebeforenow`, you will get different error messages for an invalid date and a valid date that is after now

Well, this is exactly what I want. I think it's more precise. But if you prefer a single parser and message, I'll modify them.

Anyway I think it will be more, more powerful if you could call another validator inside the validator. You should think about adding this possibility.

Reply:
Done.

Reply:
Cool. Do you know how to squash your commits?

Reply:
Mh, no. How can I do?

Reply:
If you want to learn, google is your friend. For example: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642

Reply:
Mh, I preferred to undo all and redo it. For three files...

See #1304